release-team/role-handbooks/enhancements/README.md

```diff
@@ -43,10 +43,17 @@ Helpful characteristics of an Enhancements Lead include:
 - product / project / program management experience
 - release management experience
 
-Approximate Time Commitments
-- Beginning of the cycle through enhancement freeze: 6-10 hours a week fluctuating based on how many SIG meetings need to be attended
-- Enhancement Freeze through Code Freeze: 4-7 hours a week
-- Code Freeze through Release Day: 1-4 hours a week
+### Time Commitments
+
+Enhancement is one of the most time-intensive areas of the release team, and especially so during the early parts of the release. An Enhancements Lead can expect to spend:
+- Beginning of the cycle through enhancement freeze: ~8–15 hours a week
+- Week of enhancements freeze: 20+ hours
+- Enhancement Freeze through Code Freeze: ~4–7 hours a week
+- Code Freeze through Release Day: ~1–4 hours a week
+
+Note that Enahncements Lead in particular will need to do work **during the week** during the early release, and will need to be available at least daily.
+
+Enhancements shadows have much lower time committments, but can still expect to spend 5+ hours a week during the early release, and ~1–4 hours a week after enhancements freeze. Unlike Enhancements Lead, shadows can expect to do their work largely on weekends if they desire.
```

Comment:
This is where we can really set the shadow expectation for the future cycles. IMO, shadows should plan to spend **more** than min 5 hours/week especially at the beginning of the cycle, especially a week leading up to enhancements freeze. That will help grow a better enhancements team and a better successor for future cycles as they will have more time to get embedded in the role.

IMO, we should expect 10+ hours (maybe even 15+) for shadows at the beginning of the cycle. Leads are responsible for double-checking shadows work which requires more commitment, but shadows should also be taking ownership of the assigned tasks related to the enhancements. We should clearly put that expectation here. This can also help distribute the load from Lead, allowing more flexibility of time commitment for the Lead. I just think 20+ hours for Lead and 5+ hours for shadows is not a good balance.

Maybe current/previous enhancements shadows can suggest the hours for shadows? especially @jrsapi (1.21 and 1.22 enhancements shadow), I think your feedback would be really valuable here :)

Reply:
also cc the rest of the current shadows @salaxander @reylejano @gracenng @supriya-premkumar
Feedback on the time commitment for shadows would be appreciated! thanks!

Reply:
Thanks for the good ideas @annajung. I'll survey a few recent Enhancements shadows and try to see if there's anyone who'd like to share their viewpoints. I agree we could do more to involve shadows more, but of course that requires more time commitment.

Reply:
I agree with Anna on balancing out the time commitment between lead and shadows.

Personally, the two weeks leading up to Enhancement Freeze I was putting in 10+ hours a week and that aligned with what I expected. James did an amazing job keeping us in shape by picking up enhancements that pops up here and there, so he had almost 2x the issues we do. Certainly this amount should be redistributed in the future and in 1.22 Code Freeze 😄

Reply:
I don't have data, but I think I spent about the same as an enhancements shadow in v1.21. What kind of time do we think is appropriate for the shadows? Maybe about half of what the lead has as an expectation? Or move them closer together even. We could give general time commitments for all members, and then say (the lead can expect to spend +X% more).

Reply:
I've had the experience from a previous cycle but found myself spending about 10 hours in weeks 1 and 2. Then when we got close to freeze it was about 15 hours. We should definitely raise from five.

Reply:
10-15 hours a week or two weeks before the enhancements freeze might be more realistic from what @gracenng and @jrsapi shared. By updating the doc to reflect the correct amount of time will ensure correct expectations for shadows.

Was there any increase in hours spent for code freeze? That might be also good to insert.

I don't think this is necessarily true `Enhancements shadows have much lower time committments`, rather, their work is mostly loaded in the beginning.

Reply:
I've reworked this paragraph to try to reflect the outcome of this discussion. Could folk give it another look? cc @annajung @reylejano @salaxander @jrsapi @gracenng